geode-pulse/src/test/java/org/apache/geode/tools/pulse/tests/rules/WebDriverRule.java

```diff
@@ -78,14 +93,17 @@ private void login() {
     passwordElement.submit();
 
     driver.get(getPulseURL() + "clusterDetail.html");
-    WebElement userNameOnPulsePage =
-        (new WebDriverWait(driver, 30)).until(new ExpectedCondition<WebElement>() {
+    WebElement userNameOnPulsePage;
```

Comment:
Why declare and assign on two different lines?

Reply:
cleaned out.